stan/math/prim/scal/fun/ldexp.hpp

```diff
@@ -17,7 +17,7 @@ namespace stan {
       */
     template <typename T>
     inline T ldexp(const T& a, int b) {
-      return a * std::pow(2.0, b);
+      return a * exp2(b);
```

Comment:
To run the header tests, use `make test-math-dependencies`, right? When I corrected for the error message it gave me, the next time I tried to ran it, 'nothing to be done...' Is this an expected behavior?